optional/source/how-to-do-a-task-1.html.md.erb

```diff
@@ -0,0 +1,36 @@
+---
+title: How to [name of task]
```

Comment:
Keeping three duplicates of this page template seems like additional maintenance burden we could avoid; could we instead just tell users that they can copy this template as needed?

Reply:
Hi @lfdebrux - good suggestion, thank you! I've updated that now. Could you review and approve if you're happy?